Simplified/NYPLRemoteViewController.h

```diff
@@ -22,11 +21,16 @@
 - (id)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil NS_UNAVAILABLE;
 
-/// |handler| may not be nil. |handler| is strongly retained. |data| will never be nil as the handler
-/// is only called if the data was downloaded successfully. The handler may return nil to indicate
-/// that there is something wrong with the data.
+/**
+ This is the designated initializer.
+ @param URL The URL where to fetch the data at.
+ @param handler Must not be nil. It is strongly retained. The @p data parameter
+ will never be @p nil as this handler is only called if the download was
+ successful. This may return nil to indicate that there's something wrong
+ with the downloaded data.
+ */
```

Comment:
Is `@p` specific to some markup system?

Reply:
yep, it's the objc header-doc formatting :)

Reply:
it just highlights the following word.